### PR TITLE
Create setting-up-users-instance-permissions.liquid

### PR DESCRIPTION
**Pull request by diana-lakatos**
Setting Up a User's Instance Permissions tutorial for the Get Started/Partner Portal section.

### CODE REVIEW

File: marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+slug: get-started/partner-portal/setting-up-users-instance-permissions
+searchable: true
+---
+This guide will help you give Instance permissions to users of the Partner Portal. Permissions include:
+
+* Being allowed to create new Instances 
+* Having read, write, or write and deploy access to different Instances 
+
+## Requirements
+To be able to set up Partner permissions for users on the Partner Portal, you have to have access to our Partner Portal, and the users you would like to give permissions to.
```
Comment (kv109) [inline marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid]: "Partner" => "Instance"

File: marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+To grant permissions to access Instances, select the Instances you would like to give the user access to by checking their checkboxes.
+
+Select access level (`Read`, `Write`, or `Write and deploy`):
+* `Read`: The user can see the Instance, but can't modify it. 
+* `Write`: The user can see and modify the Instance, but can't deploy. E.g. the user can change configuration, but can't make changes in the codebase and deploy them.  
+* `Write and deploy`: The user can see, modify, and deploy to the Instance. 
+
+Once you've selected the permissions, click on `Submit`.
+
+You can check the user's permissions anytime under `This user has access to following instances` on the `Instances` tab of the user's information page.
```
Comment (kv109) [inline marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid]: "information" page, hmm, maybe "user permissions" page? I'm not sure.

**Review by kv109 — CHANGES_REQUESTED** [review]

File: marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid (hunk 1 above)
Comment (diana-lakatos) [reply marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid]: @kv109 Thank you! Fixed.

File: marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid (hunk 2 above)
Comment (diana-lakatos) [reply marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid]: This one is before you get to the "user permissions" page, so I wouldn't call it that. Maybe "the selected user's page"? Would that be better?
I mean this page for example: https://portal.apps.near-me.com/users/8 (with different bits of information about the user, like email, Partner, etc.)

**Comment by diana-lakatos** [comment]
@kv109 Edited based on our discussion. Thanks!

**Review by kv109 — APPROVED** [review]

File: marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+Select access level (`Read`, `Write`, or `Write and deploy`):
+* `Read`: The user can see the Instance, but can't modify it. 
+* `Write`: The user can see and modify the Instance, but can't deploy. E.g. the user can change configuration, but can't make changes in the codebase and deploy them.  
+* `Write and deploy`: The user can see, modify, and deploy to the Instance. 
+
+Once you've selected the permissions, click on `Submit`.
+
+You can check the user's permissions anytime under `This user has access to following instances` on the `Instances` tab. 
+
+{% include 'alert/note', content: 'Granting permissions to Partners and Instances are completely separate: If you grant permissions to an Instance, the user won't automatically get access to the Partner of that Instance. You have to set this up separately as explained in the tutorial linked below.' %}
```
Comment (pavelloz) [inline marketplace_builder/views/pages/get-started/partner-portal/setting-up-users-instance-permissions.liquid]: single quote inside single quotes

**Comment by diana-lakatos** [comment]
@pavelloz Fixed, thank you!

**Pull request merged**